README.md

```diff
@@ -56,22 +56,25 @@ And run `webpack` via your preferred method.
 
 ## Options
 
-|                   Name                    |         Type         |         Default          | Description                                            |
-| :---------------------------------------: | :------------------: | :----------------------: | :----------------------------------------------------- |
-|     **[`lessOptions`](#lessoptions)**     | `{Object\|Function}` | `{ relativeUrls: true }` | Options for Less.                                      |
-|  **[`additionalData`](#additionalData)**  | `{String\|Function}` |       `undefined`        | Prepends/Appends `Less` code to the actual entry file. |
-|       **[`sourceMap`](#sourcemap)**       |     `{Boolean}`      |    `compiler.devtool`    | Enables/Disables generation of source maps.            |
-| **[`webpackImporter`](#webpackimporter)** |     `{Boolean}`      |          `true`          | Enables/Disables the default Webpack importer.         |
-|  **[`implementation`](#implementation)**  |  `{Object\|String}`  |          `less`          | Setup Less implementation to use.                      |
+- **[`lessOptions`](#lessoptions)**
+- **[`additionalData`](#additionalData)**
+- **[`sourceMap`](#sourcemap)**
+- **[`webpackImporter`](#webpackimporter)**
+- **[`implementation`](#implementation)**
 
 ### `lessOptions`
 
-Type: `Object|Function`
+Type:
+
+```ts
+type lessOptions = object | ((loaderContext: LoaderContext) => object})
```

Comment:
This one should be   `((content: string, loaderContext: LoaderContext) => string);` ? In the examples the first param seems to be loaderContext?

https://github.com/webpack-contrib/less-loader/blob/059e8967ee6638c5939d13a457d8ec24fc9d241a/README.md?plain=1#L124

Reply:
Based on code https://github.com/webpack-contrib/less-loader/blob/master/src/utils.js#L165, it should be `((loaderContext: LoaderContext) => lessOptions)`, there is `lessOptions` is options for less, we can use `import("less").Options` (need to check real names)

Reply:
Changed it. it was `options` not `Options`